spec/graphql/resolvers/invoice_resolver_spec.rb

```diff
@@ -93,6 +95,28 @@
     expect(group['value']).to eq('usa')
   end
 
+  context 'when invoice has credit notes' do
+    before do
+      create(:credit_note, invoice: invoice)
+    end
+
+    it 'returns an error' do
+      result = execute_graphql(
+      current_user: membership.user,
```

Comment:
Layout/FirstArgumentIndentation: Indent the first argument one step more than the start of the previous line.